### PR TITLE
Bm edit copy

### PR DESCRIPTION
**Pull request by bamandel**

### CODE REVIEW

File: src/main/java/com/synopsys/integration/alert/web/user/RoleController.java [hunk 1]
```diff
@@ -73,6 +74,20 @@ public RoleController(ContentConverter contentConverter, ResponseFactory respons
         return responseFactory.createOkContentResponse(contentConverter.getJsonString(roleActions.getRoles()));
     }
 
+    @PutMapping
+    public ResponseEntity<String> updateRole(@RequestBody RolePermissionModel rolePermissionModel) {
+        if (!hasPermission(authorizationManager::hasCreatePermission)) {
```
Comment (gkillough) [inline src/main/java/com/synopsys/integration/alert/web/user/RoleController.java]: Should this be checking `create`? Do we have another permission type?

File: src/main/js/util/configurationRequestBuilder.js [hunk 2]
```diff
@@ -71,6 +71,19 @@ export function createUpdateRequest(apiUrl, csrfToken, configurationId, fieldMod
     });
 }
 
+export function createUpdateWithoutIdRequest(apiUrl, csrfToken, model) {
```
Comment (gkillough) [inline src/main/js/util/configurationRequestBuilder.js]: Do we validate that the role name is unique on the back-end? If not, we should.

Comment (bamandel) [reply src/main/js/util/configurationRequestBuilder.js]: We do :)

File: src/main/java/com/synopsys/integration/alert/web/user/RoleController.java (hunk 1 above)
Comment (bamandel) [reply src/main/java/com/synopsys/integration/alert/web/user/RoleController.java]: Changed to write

**Pull request merged**